### PR TITLE
Pr.libuv.excludes

### PR DESCRIPTION
**Pull request by akalsi87**

### CODE REVIEW

File: cmake/configs/default.cmake [hunk 1]
```diff
@@ -65,7 +65,7 @@ if(MSVC_VERSION LESS 1600)
     # for VS10 - version without support C++11
   hunter_config(jsoncpp VERSION 0.7.0)
 else()
-  hunter_config(jsoncpp VERSION 1.7.7)
+  hunter_config(jsoncpp VERSION 1.8.0)
```
Comment (ruslo) [inline cmake/configs/default.cmake:68]: This doesn't belong here. Only `.travis.yml`/`appveyor.yml`, see README:
* https://github.com/ingenue/hunter

**Review by ruslo — CHANGES_REQUESTED** [review]
Please squash your changes into one commit.

**Comment by ruslo** [comment]
> Please squash your changes into one commit.

There are still two commits...

**Comment by akalsi87** [comment]
I tried but it says I have merge conflicts, and that's why I had two commits last time as well. help please @ruslo

**Comment by ruslo** [comment]
> help please @ruslo

Fetch branch `pkg.libuv`, exclude toolchains and push changes. What kind of problems can appear here? It's kind of trivial...

**Pull request closed without merging**